docs/AddUnitTests.md

```diff
@@ -48,14 +48,49 @@ will just tell you that `TryGetValue` failed, without telling you the key it was
 
 The output will now tell you that `IsTrue` failed, but also why. This is especially handy in cases where your `Assert` statement appears within a loop
 
-### Working with Fakes
-Due to the known issue (overwriting fakes assemblies from different unit tests if they have same name), all `Fakes` assemblies are generated in a single project (`AccessibilityInsights.Fakes.Prebuild`). 
+### Working with Microsoft Fakes
 
-If you need to add any new `Fake` assembly, please follow the below steps:
+#### Adding a test assembly that requires Fakes
+
+Due to the known issue (overwriting fakes assemblies from different unit tests when they have the same name), all `Fakes` assemblies are generated in a single project (`AccessibilityInsights.Fakes.Prebuild`). 
+
+If you need to add a new `Fake` assembly, please follow the below steps:
 1. Add a new fakes file into `AccessibilityInsights.Fakes.Prebuild`. 
 2. Build `AccessibilityInsights.Fakes.Prebuild`.
 3. Add a project dependency (not project reference) of `AccessibilityInsights.Fakes.Prebuild` into the unit test which needs a new fakes assembly.
 4. Add an assembly reference of *.fakes.dll (or exe) in `AccessibilityInsights.Fakes.Prebuild/FakesAssemblies` folder to the unit test.
 5. Add `Microsoft.QualityTools.Testing.Fakes.dll` into the assembly reference of the unit test.
+6. Any references to Fakes assemblies which are added to any test projects should be conditional on the environment variable `FAKES_SUPPORTED = 1`. See the next section for details.
+
+#### Important: Not all editions of Visual Studio support Fakes
```

Comment:
my 2c - Should we perhaps move this up in the order. This way they will get this heads up before they read through the steps / try to follow them step by step.

Reply:
I think the order is okay. My feeling is that it's all under the heading of working with Fakes and the user will read the entire section. Perhaps I will put in a note to make sure people read the entire section before making any changes.